main.go

```diff
@@ -106,22 +107,23 @@ func main() {
 	}
 	version.PrintVersionLogs(setupLog)
 
-	syncDuration := time.Duration(syncPeriodSeconds) * time.Second
+	// syncDuration := time.Duration(syncPeriodSeconds) * time.Second
 	restConfig := ctrl.GetConfigOrDie()
 	restConfig.UserAgent = "wpa-controller"
 	if clientTimeoutDuration != 0 {
 		// override client timeout duration if set
 		restConfig.Timeout = clientTimeoutDuration
 	}
 	mgr, err := ctrl.NewManager(restConfig, config.ManagerOptionsWithNamespaces(setupLog, ctrl.Options{
-		Scheme:                     scheme,
-		MetricsBindAddress:         fmt.Sprintf("%s:%d", host, metricsPort),
-		Port:                       9443,
+		Scheme: scheme,
+		Metrics: metricsserver.Options{
+			BindAddress: fmt.Sprintf("%s:%d", host, metricsPort),
+		},
 		LeaderElection:             enableLeaderElection,
 		LeaderElectionID:           "watermarkpodautoscaler-lock",
 		LeaderElectionResourceLock: leaderElectionResourceLock,
 		HealthProbeBindAddress:     fmt.Sprintf("%s:%d", host, healthPort),
-		SyncPeriod:                 &syncDuration,
+		// SyncPeriod:                 &syncDuration,
```

Comment:
does it means that this option was removed?

Reply:
This option was moved. I’ve updated this piece of code with the new option location.